uiowa/uiowa
# [Find Text] Update version, remove SiteNow Find Text

Resolves #6552 (first steps--we should create a followup for further protection through paging or other solutions)
Resolves #5383 





# How to test


`composer install`
Sync a site, use the Find Text tool
Try searching for an empty string and/or a string of fewer than 3 characters
See that not only do you get the error message, but also it doesn't bring up results (or attempt to bring up results and crash the page)
When searching a string, try hitting Enter while the search text field is in focus, and see that it successfully submits the form - Huzzah!
